capstone/src/main/java/com/google/sps/Objects/Option.java

```diff
@@ -5,22 +5,22 @@
 public final class Option {
 
   private final String text;
-  private ArrayList<Integer> votes;
+  private ArrayList<String> userVotes;
 
   public Option(String text) {
     this.text = text;
-    this.votes = new ArrayList<Integer>();
+    this.userVotes = new ArrayList<String>();
   }
 
   public String getText() {
     return this.text;
   }
 
-  public void addVote(int id) {
-    this.votes.add(id);
+  public void addVote(String id) {
```

Comment:
`addVote(String id)` On its own, is not immediately clear.

I would recommend renaming the parameter `userId`